exporter/fileexporter/file_exporter.go

```diff
@@ -93,6 +113,20 @@ func exportMessageAsLine(e *fileExporter, buf []byte) error {
 	return nil
 }
 
+func exportMessageAsBuffer(e *fileExporter, buf []byte) error {
+	// Ensure only one write operation happens at a time.
+	e.mutex.Lock()
+	defer e.mutex.Unlock()
+	// write the size of each message before writing the message itself.
```

Comment:
Is this a standard format? If so, let's link to it in the readme. Otherwise, we should add some language to the readme that explains to users how to interpret the file.

Reply:
We cannot determine where a message ends on their own, so `Proto buffers` Guide says '[The easiest way to solve this problem is to write the size of each message before you write the message itself. '](https://developers.google.com/protocol-buffers/docs/techniques#streaming)
I will add in the comments.